hawkular-inventory-api/src/main/java/org/hawkular/inventory/api/model/Entity.java

```diff
@@ -72,27 +74,89 @@ public final String toString() {
      * the subclasses are free to use a blueprint type not inheriting from this one.
      */
     public abstract static class Blueprint extends AbstractElement.Blueprint {
-        @XmlAttribute
         private final String id;
+        private final Map<String, Set<CanonicalPath>> outgoing;
```

Comment:
So if I understand it correctly, one can specify the relationships in the entity blueprint itself and it will be automatically wired when creating this entity. I.e. 1 call instead of N (?)